src/extension/src/MsftLinuxPatchExtShim.sh

```diff
@@ -36,7 +36,7 @@ function find_python(){
     local python_exec_command=$1
 
     # Check if there is python defined.
-    for p in python3 python python2 /usr/libexec/platform-python; do
+    for p in python3 python python2 /usr/libexec/platform-python /usr/share/oem/python/bin/python3 /usr/share/oem/python/bin/python; do
```

Comment:
We'll need to add these and "/usr/libexec/platform-python" in /extension/src/ProcessHandler.py --> get_python_cmd() also, for Extension to invoke Core successfully

Reply:
Are we really missing 2 existing entries (last 2) + the 2 new ones there??

Reply:
Updated to auto-discover from manual discovery. Manual discovery is only a fallback for now. To be removed later.